### PR TITLE
Fixed the illegal string offset warning

### PR DESCRIPTION
**Pull request by andrasguseo**
Added a check to make sure the right type is passed when creating shortcode.
Originally $atts would come empty, which is a string. In this case it needs to be converted to array.

Based on:
https://codex.wordpress.org/Shortcode_API#Overview
"$atts - an associative array of attributes, or an empty string if no attributes are given"

Central ticket:
https://central.tri.be/issues/115913

### CODE REVIEW

**Review by cliffordp — CHANGES_REQUESTED** [review]
Please change `if ( is_string( $atts ) )` to `if ( ! is_array( $atts ) )` since we know we'll be acting on the array. Good job.

File: readme.txt [hunk 1]
```diff
@@ -37,6 +37,10 @@ We're always interested in your feedback and our [premium forums](https://theeve
 
 == Changelog ==
 
+= [2.1.2] 2018-11-16 =
```
Comment (cliffordp) [inline readme.txt]: Correct the date

File: tribe-ext-list-venues-organizers-shortcodes.php [hunk 2]
```diff
@@ -259,13 +259,29 @@ protected function format() {
 			wp_reset_postdata();
 		}
 
+		/**
+		 * @param $atts - an associative array of attributes, or an empty string if no attributes are given
```
Comment (cliffordp) [inline tribe-ext-list-venues-organizers-shortcodes.php:274]: Add summary line to docblock

File: tribe-ext-list-venues-organizers-shortcodes.php [hunk 3]
```diff
@@ -259,13 +259,29 @@ protected function format() {
...
+			if ( ! is_array( $atts ) ) {
+				$atts = [];
+			}
 			$atts['post_type'] = Tribe__Events__Venue::POSTTYPE;
 
 			return $this->do_base_shortcode( $atts );
 		}
 
+		/**
+		 * @param $atts - an associative array of attributes, or an empty string if no attributes are given
```
Comment (cliffordp) [inline tribe-ext-list-venues-organizers-shortcodes.php:290]: Add summary line to docblock

**Review by cliffordp — CHANGES_REQUESTED** [review]

File: tribe-ext-list-venues-organizers-shortcodes.php (hunk 2 above)
Comment (andrasguseo) [reply tribe-ext-list-venues-organizers-shortcodes.php:274]: Done

File: readme.txt (hunk 1 above)
Comment (andrasguseo) [reply readme.txt]: Done

File: tribe-ext-list-venues-organizers-shortcodes.php (hunk 3 above)
Comment (andrasguseo) [reply tribe-ext-list-venues-organizers-shortcodes.php:290]: Done

**Review by cliffordp — APPROVED** [review]

**Pull request merged**